Simplenote/src/androidTest/java/com/automattic/simplenote/integration/TagsActivityTest.kt

```diff
@@ -0,0 +1,226 @@
+package com.automattic.simplenote.integration
+
+import android.widget.EditText
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.MediumTest
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.automattic.simplenote.BaseUITest
+import com.automattic.simplenote.R
+import com.automattic.simplenote.TagsActivity
+import com.automattic.simplenote.models.Note
+import com.automattic.simplenote.models.Tag
+import com.automattic.simplenote.utils.isToast
+import com.automattic.simplenote.utils.withRecyclerView
+import org.hamcrest.CoreMatchers
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4ClassRunner::class)
+@MediumTest
+class TagsActivityTest : BaseUITest() {
+
+    inner class TagAndCounter(val tag: Tag, val counter: String)
+    inner class TestData(val tags: List<TagAndCounter>, val notes: List<Note>)
+
+    private fun launchTagsActivityWithTestData(): TestData {
```

Comment:
This is very minor but could you please move this method to the end of this class?

Reply:
Fixed in ec95886df25799bf0d8e652599c61d1321f79266